### ISSUE TITLE
Mac OSX: "sudo: unknown user: postgres"

### ISSUE THREAD

**Issue by Vincent14** [sleede/fab-manager#34]
In [this](https://github.com/LaCasemate/fab-manager#setup-fabmanager-in-postgresql) documentation section, I had to do:

```
$ brew install postgres
$ pg_ctl -D /usr/local/var/postgres -l /usr/local/var/postgres/server.log start
$ sudo -i -u postgres
```

And got this:

```
sudo: unknown user: postgres
```

I found the solution [here](http://stackoverflow.com/questions/16689934/sudo-unknown-user-postgres):

```
sudo psql -U my_macosx_username postgres
```

**Comment by sylvainbx** [comment]
Hi, thanks for reporting this detail, it could be usefull to know!

I do not use Mac personnally, but other people in the team do. I'll wait for their feedback on how they managed this problem before updating the documentation.

**Comment by sylvainbx** [comment]
This was fixed in commit referenced above on the [dev](https://github.com/LaCasemate/fab-manager/tree/dev) branch. These changes will be released with next stable version (probably 2.4.0).
